cli/cmd/plugin_publish.go

```diff
@@ -83,6 +82,7 @@ func runPluginPublish(ctx context.Context, cmd *cobra.Command, args []string) er
 			return errors.New("invalid plugin name. Must be in format <team_name>/<plugin_name>")
 		}
 		teamName, pluginName = parts[0], parts[1]
+		pkgJSON.Team, pkgJSON.Name = teamName, pluginName
```

Comment:
This is backwards compatibility code from when we had `plugin publish <team>/<name>` instead of inside `package.json`.

I cleaned up some of the arguments to `UploadPluginBinary` to pass only the package JSON so we need to set these here